README.md

```diff
@@ -210,6 +212,13 @@ class NewProvider(override val hooks: List<Hook<*>>, override val metadata: Meta
         // add necessary changes on context change
     }
 
+    override fun observe(): Flow<OpenFeatureEvents> {
+        // return the observation flow
+    }
```

Comment:
This is a Kotlin specific solution that is not on the Spec (i.e. the SDK listening on Events from the Provider in order to perform some logic). Shall we document here what are the expected Events that every Provider will have to implement?

Reply:
PR has been opened for a while, let's prioritize getting this one merge and think of more improvements in future PRs